Neos.FluidAdaptor/Tests/Unit/Core/Widget/AjaxWidgetMiddlewareTest.php

```diff
@@ -200,67 +189,37 @@ public function handleCancelsComponentChainIfWidgetContextIsPresent()
     {
         $mockWidgetId = 'SomeWidgetId';
         $mockControllerObjectName = 'SomeControllerObjectName';
-        $this->mockHttpRequest->expects(self::any())->method('getParsedBody')->willReturn([
-            '__widgetId' => $mockWidgetId,
-        ]);
-
-        $mockWidgetContext = $this->getMockBuilder(WidgetContext::class)->getMock();
-        $mockWidgetContext->expects(self::atLeastOnce())->method('getControllerObjectName')->will(self::returnValue($mockControllerObjectName));
-        $this->mockAjaxWidgetContextHolder->expects(self::atLeastOnce())->method('get')->with($mockWidgetId)->will(self::returnValue($mockWidgetContext));
-        $mockActionRequest = $this->getMockBuilder(ActionRequest::class)->disableOriginalConstructor()->getMock();
-        $this->mockActionRequestFactory->expects(self::any())->method('prepareActionRequest')->willReturn($mockActionRequest);
-        $this->mockComponentContext->expects(self::any())->method('getParameter')->willReturnMap([
-            [RoutingComponent::class, 'matchResults', []],
-            [DispatchComponent::class, 'actionRequest', $mockActionRequest]
-        ]);
-        $this->mockComponentContext->expects(self::any())->method('setParameter')->with(ComponentChain::class, 'cancel', true);
-
-        $this->ajaxWidgetComponent->handle($this->mockComponentContext);
-    }
-
-    /**
-     * @test
-     */
-    public function handleInjectsActionRequestToSecurityContext()
```

Comment:
While this test would have failed with the removal of `securityContext->setRequest()` from above, I wonder why nothing else fails.